src/main/java/projectI/Parser/Parser.java

```diff
@@ -69,6 +77,12 @@ private SimpleDeclarationNode tryParseSimpleDeclaration(int begin, int endExclus
         return tryParseTypeDeclaration(begin, endExclusive);
     }
 
+    /**
+     * parse a variable declaration
```

Comment:
I would prefer comments to begin with capital letters, but it is not that important.